orca-front50/src/test/groovy/com/netflix/spinnaker/orca/front50/pipeline/EnabledPipelineValidatorSpec.groovy

```diff
@@ -121,7 +121,7 @@ class EnabledPipelineValidatorSpec extends Specification {
     execution = pipeline {
       application = "whatever"
       pipelineConfigId = "1337"
-      trigger = new PipelineTrigger(null, [strategy: true])
+      trigger = new PipelineTrigger(pipeline {}, [strategy: true])
```

Comment:
Running these tests in IntelliJ could fail if compiler `@Nonnull` checks are enforced.